vendor/bat-native-ledger/src/bat_client.cc

```diff
@@ -506,32 +531,46 @@ void BatClient::setGrantCallback(bool success,
   ledger_->OnGrantFinish(ledger::Result::LEDGER_OK, grant);
 }
 
-void BatClient::getGrantCaptcha() {
+void BatClient::getGrantCaptcha(bool isAndroid) {
```

Comment:
rather than pass `isAndroid` (which should is `is_android`), it seems like we should pass the prefix version and keep the decision about when to use it in brave_rewards

Reply:
and actually why are we using a different version for android anyway?

Reply:
@bridiver endpoints for attestation and actual captcha are different

Reply:
If you about naming I just followed the pattern of other variables in the class `captchaResponse`, `promotionId` etc

Reply:
it isn't about naming, I just think it's better within ledger to pass what we want (the version) vs the platform we are using in our particular implementation in brave-core